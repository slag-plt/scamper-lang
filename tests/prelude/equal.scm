(equal? 4 4)
(equal? 10 20)
(equal? "Hello" "Hello")
(equal? "Hello" "HELLO")
(equal? 4 "4")
(equal? 4 4.0)