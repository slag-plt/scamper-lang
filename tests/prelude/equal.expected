#t
#f
#t
#f
#f
#t
